extensions/amp-o2-player/0.1/validator-amp-o2-player.protoascii

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-o2-player
+  tag_name: "script"
```

Comment:
Due to changes since you started this PR, tag_name fields are now upper case. Can you please change this to:
`tag_name: "SCRIPT"`